.gitignore

```diff
@@ -36,10 +36,9 @@ site/
 # macOS folder metadata
 .DS_Store
 
-# Temporary integrtion test files
+# Temporary integration test files
 tmp_*
 .coverage
-.coverage*
```

Comment:
This line is not necessary and covers up any changes to .coveragerc configuration.